vignettes/how_to_harmonize.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "How to harmonize across survey cycles"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  eval = F,
+  echo = T,
+  results = "hide",
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Introduction 
+
+This vignette explains how you can transform variables across multiple CCHS datasets using the full datasets to the _cchsflow_ package. The full PUMF datasets can be found [here](https://odesi.ca/). A full harmonized dataset of all cchsflow variables
+can be found [here](https://osf.io/j5wgu). With the original PUMF datasets, data should be renamed such that it specifies the survey and cycle year, which follows the format of the _p sample data (ex. cchs2001, cchs2013_2014).
+
+Note: Harmonizing cycles before 2014 with cycles from 2015 onward is not advised as Statistics Canada has made major survey design changes.
+
+## How to combine a single variable across multiple cycles

```

Comment:
I wonder if we should add a little text before this section that gives a quick introduction the function we will be using?